### DISCUSSION TITLE
velocity kmz file

### DISCUSSION THREAD

**Discussion by jaysonowh** [insarlab/MintPy#1213] [Q&A]
@yunjunz @asjohnston-asf 
I finished running Mintpy, but when I opened velocity kmz file this is what I got.
can you tell me what's the problem?

![image](https://github.com/insarlab/MintPy/assets/171894510/d723f44f-dd82-4c39-9b7a-6fefa4ed1751)

**Comment by yunjunz** [comment]
https://github.com/insarlab/MintPy/pull/1210

**Reply by jaysonowh** [reply]
> #1210

I updated MintPy but it is still the same
